backend/scrapers/police_coders_groups/group_count.py

```diff
@@ -0,0 +1,101 @@
+'''
+Intended purpose of this script:
+1. Get LA and LSOA boundaries as shape objects
+2. Get list of groups and their coordinates
+3. Locate groups to LA and LSOA boundaries
+4. Count how many groups are in each LA & LSOA
+5. Save LA and LSOA counts into seperate csvs
+
+'''
+
+import pandas as pd
+from pandas import Series
```

Comment:
As you've already imported `pandas as pd` you don't need to import Series separately. I've left some comments with suggestions on how to use the same functions is a most typical pandas way - don't have to implement them now necessarily but just in case it's helpful!